example.env

```diff
@@ -12,6 +12,9 @@ FLUENCE_ENV="local"
 # To turn on Countly debugger
 DEBUG_COUNTLY="true"
 
+# To set path for the user .fluence dir
+# USER_FLUENCE_DIR_PATH="/absolute/path/to/users/.fluence"
```

Comment:
```suggestion
# FLUENCE_DIR="/absolute/path/to/users/.fluence"
```
Let's name it like this? Analogous to `IPFS_PATH` of IPFS.

Reply:
I think USER_FLUENCE_DIR_PATH is more explicit. There is no reason to save characters here IMHO